_sections/act_and_prepare/en/02-do/16-wear-a-mask-when-in-public.md

```diff
@@ -10,6 +10,8 @@ The viron (complete virus particle) that causes COVID-19 is roughly the same siz
 
 On April 3, the US Center for Disease Control (CDC), "recommends wearing cloth face coverings in public settings where other social distancing measures are difficult to maintain (e.g., grocery stores and pharmacies) **especially** in areas of significant community-based transmission". See the [article on their site](https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/cloth-face-cover.html)
 
+On April 4, the World Health Organization's top emergencies expert, Mike Ryan, said at a news conference: "... The idea of using respiratory coverings or mouth coverings to prevent coughing or sneezing projecting disease into the environment and towards others ... that in itself is not a bad idea". There's no page reflecting this change of policy yet.
```

Comment:
```suggestion
On April 4, the World Health Organization's top emergencies expert, Mike Ryan, said at a news conference: "... The idea of using respiratory coverings or mouth coverings to prevent coughing or sneezing projecting disease into the environment and towards others ... that in itself is not a bad idea". There's no page reflecting this change of policy yet, [but the quote is in Reuters](https://www.reuters.com/article/us-health-coronavirus-who/who-opens-door-to-broader-use-of-masks-to-limit-spread-of-coronavirus-idUSKBN21L2UK).
```